### PR TITLE
Side scroller redux

### PR DESCRIPTION
**Pull request by davidthecomposer**
some fixes to avoid linting errors.

### CODE REVIEW

File: SideScroller.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ interface SideScrollerProps {
    */
   disableTouchscreenMode?: boolean
 
-  setContainerAnimation?: (animation: gsap.core.Tween) => void
+  setContainerAnimation?: (animation: gsap.core.Tween) => void | undefined
```
Comment (RJWadley) [inline SideScroller.tsx:21]: did you mean `((animation: gsap.core.Tween) => void) | undefined`?

File: SideScroller.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default function SideScroller({
   children,
   ease = "none",
   disableTouchscreenMode = false,
-  setContainerAnimation = () => {},
+  setContainerAnimation = undefined,
```
Comment (RJWadley) [inline SideScroller.tsx:31]: useless undefined

File: SideScroller.tsx [hunk 3]
```diff
@@ -18,7 +18,7 @@ interface SideScrollerProps {
    */
   disableTouchscreenMode?: boolean
 
-  setContainerAnimation?: (animation: gsap.core.Tween) => void
+  setContainerAnimation?: (animation: gsap.core.Tween) => void | undefined
   /**
    * allows adds the capability to add the timeline to another animation so that the containerAnimation property can be set on that scrolltrigger.
```
Comment (RJWadley) [inline SideScroller.tsx:23]: fix this jsdoc while you're here

File: SideScroller.tsx (hunk 1 above)
Comment (RJWadley) [inline SideScroller.tsx:21]: don't use void, try typing this as `(animation: gsap.core.Tween) => unknown`

Comment (davidthecomposer) [reply SideScroller.tsx]: added unknown

**Review by RJWadley — APPROVED** [review]

**Pull request merged**